apps/preview/next/pages/showcases/MegaMenu/OpenOnClick.tsx

```diff
@@ -0,0 +1,263 @@
+      <header className="flex justify-center w-full border-0 bg-primary-700 border-neutral-200 h-14 md:relative md:h-20 md:z-10">
+        <div className="flex items-center flex-nowrap justify-start h-full max-w-[1536px] w-full px-4 md:px-10">
+          <a href="/" aria-label="SF Homepage" className="inline-block text-white mr-2 lg:mr-10">
+            <img src={brandLogo.src} alt="Sf Logo" className="hidden lg:block mr-10 w-[12.5rem] h-[1.75rem]" />
+            <img src={brandLogoSign.src} alt="Sf Logo" className="hidden w-10 h-10 mr-4 md:block lg:hidden" />
+            <img src={brandLogoSign.src} alt="Sf Logo" className="block w-8 h-8 mr-4 min-w-8 md:hidden" />
+          </a>
+          <nav className="flex w-full justify-between flex-nowrap" aria-label="SF Navigation">
+            <ul>
+              <li role="none">
+                <SfButton
+                  className="block !px-2 mr-auto text-white font-body bg-transparent hover:bg-primary-800 hover:text-white active:bg-primary-900 active:text-white"
+                  type="button"
+                  aria-haspopup="true"
+                  aria-expanded={open}
+                  slotSuffix={
+                    <>
+                      <SfIconChevronRight className="hidden rotate-90 md:inline-flex" />
+                      <SfIconMenu className="inline-flex md:hidden" />
+                    </>
+                  }
+                  variant="tertiary"
+                  onClick={() => {
+                    setOpen(true);
+                  }}
+                >
+                  <span className="hidden md:inline-flex">Categories</span>
+                </SfButton>
+                <CSSTransition
+                  ref={nodeRef}
+                  in={open}
+                  timeout={500}
+                  unmountOnExit
+                  classNames={{
+                    enter: '-translate-x-full md:opacity-0 md:translate-x-0',
+                    enterActive: 'translate-x-0 md:opacity-100 transition duration-500 ease-in-out',
+                    exitDone: 'translate-x-0 md:opacity-100',
+                    exitActive: '-translate-x-full md:opacity-0 transition duration-500 ease-in-out',
+                  }}
+                >
+                  <SfDrawer
+                    open={open}
+                    onClose={() => setOpen(false)}
+                    placement="top"
+                    className="grid grid-cols-1 gap-6 md:grid-cols-4 bg-white max-w-xs shadow-lg p-0 !fixed md:!absolute md:!top-[5rem] md:max-w-full md:p-8"
+                  >
+                    <div className="flex items-center justify-between p-4 bg-primary-700 md:hidden">
+                      <div className="flex items-center typography-text-lg font-medium text-white">Categories</div>
+                      <SfButton
+                        square
+                        variant="tertiary"
+                        aria-label="Close drawer"
```

Comment:
Yes, it is a drawer in general, but I think we could be more specific in this case. We could say "close menu", or even "close navigation menu", wdyt? I think saying just "close drawer" feels like not enough.